### DISCUSSION TITLE
请教一下条件配置

### DISCUSSION THREAD

**Discussion by Lonely233233** [hooke007/mpv_PlayKit#291] [mpv 通用]
[ontop_playback]
 profile-desc = 基于播放状态的自动置顶
 profile-cond = not pause
 profile-restore = copy
 ontop = yes
 比如这个profile-cond = not pause这里可以添加多个条件吗

**Comment by hooke007** [comment]
当然可以 https://hooke007.github.io/unofficial/mpv_profiles.html

而且下面不就是个多条件的例子？
https://github.com/hooke007/MPV_lazy/blob/0bc2fb479d6ff32175189a6fb0b88d888daf88d2/portable_config/profiles.conf#L32

**Reply by Lonely233233** [reply]
谢谢

**Reply by Lonely233233** [reply]
有了这个我就可以添加一些网站让他自动走系统代理了
